### ISSUE TITLE
Crash when trying to pile up stone bricks

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#937]
**Game Version:** 1.15.0-pre2
**Platform:** Windows
**Modded:** StepUp

### Description
Crash when trying to pile up stone bricks

### How to reproduce
1. Get a stone brick (any)
2. Try to pile up it on any surface
3. It crashes

### Expected behavior
It make ground storage of stone bricks

### Logs
```
Version: v1.15.0-pre.2 (Unstable)05/29/2021 21:27:41: Critical error occurred
System.ArgumentNullException: Value cannot be null.
Parameter name: Unable to find a texture for texture code 'material'. Giving up. Sorry.
   at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShapeElement(Int32 indexForLogging, MeshData meshdata, ShapeElement element, TesselationMetaData meta)
   at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShapeElements(MeshData meshdata, ShapeElement[] elements, TesselationMetaData meta)
   at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape(Shape shapeBase, MeshData& modeldata, Vec3f wholeMeshRotation, Vec3f wholeMeshOffset, TesselationMetaData meta)
   at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape(String typeForLogging, Shape shapeBase, MeshData& modeldata, ITexPositionSource texSource, Vec3f wholeMeshRotation, Int32 generalGlowLevel, Byte climateColorMapId, Byte seasonColorMapId, Nullable`1 quantityElements, String[] selectiveElements)
   at Vintagestory.GameContent.BlockEntityGroundStorage.getOrCreateMesh(ItemSlot itemSlot, Vec3f offset)
   at Vintagestory.GameContent.BlockEntityGroundStorage.updateMeshes()
   at Vintagestory.GameContent.BlockEntityGroundStorage.OnPlayerInteract(IPlayer player, BlockSelection bs)
   at Vintagestory.GameContent.BlockGroundStorage.CreateStorage(IWorldAccessor world, BlockSelection blockSel, IPlayer player)
   at Vintagestory.GameContent.CollectibleBehaviorGroundStorable.Interact(ItemSlot itemslot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, Boolean firstEvent, EnumHandHandling& handHandling, EnumHandling& handling)
   at Vintagestory.API.Common.CollectibleObject.<>c__DisplayClass88_0.<OnHeldInteractStart>b__0(CollectibleBehavior bh, EnumHandling& hd)
   at Vintagestory.API.Common.CollectibleObject.WalkBehaviors(CollectibleBehaviorDelegate onBehavior, Action defaultAction)
   at Vintagestory.API.Common.CollectibleObject.OnHeldInteractStart(ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, Boolean firstEvent, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel, EnumHandInteract useType, EnumHandHandling& handling)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseActiveSlotItem(BlockSelection blockSel, EntitySelection entitySel)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _xArlFw5lpThzIY0BN36UerAlHzU._14ww1iwrPaPwPpAZqZCUdf3d4Sm(Single )
   at _uFwaliVbJcYmzlP3dwic7HSHMr2._EuQCmN4cQf8FBHYrlJxd8EhIdNnA(Single )
   at _uFwaliVbJcYmzlP3dwic7HSHMr2._HlDCi3DfsJHaHE4jniBarrjFlHLB(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _EKato5MKvGk9ELCMKyl4BGVO7pe._ninJWCWaWyMG9ZSA1Eyytms1G5F(_BjQ4x8rTRChySIeXrK1HK1YSqec , String[] )
   at _btwtpE4G1MfBqD34njKFCvBcSRd._ninJWCWaWyMG9ZSA1Eyytms1G5F(ThreadStart )
```

**Comment by Craluminum2413** [comment]
Fixed in 1.15.0-pre.4

**Comment by Craluminum2413** [comment]
How did you fix this issue? I'm making mod that adds new piles and get the same crash on placing item
https://github.com/Craluminum2413/MorePiles/issues/1
